File: get_urls.js
const fs = require('fs');
const urls = ['https://ibb.co/vx2SwyLV', 'https://ibb.co/hJTvKs26', 'https://ibb.co/sZzNgQ5', 'https://ibb.co/Y44L5FH0'];
Promise.all(urls.map(u => fetch(u).then(r => r.text()).then(t => {
    const match = t.match(/<meta property=\"og:image\" content=\"([^\"]+)\"/);
    return match ? match[1] : '';
}))).then(links => {
    fs.writeFileSync('urls.txt', links.join('\n'));
});

File: get_urls.py
import urllib.request, re, ssl
ctx = ssl.create_default_context(); ctx.check_hostname = False; ctx.verify_mode = ssl.CERT_NONE
urls = ['https://ibb.co/vx2SwyLV', 'https://ibb.co/hJTvKs26', 'https://ibb.co/sZzNgQ5', 'https://ibb.co/Y44L5FH0']
with open('urls.txt', 'w') as f:
    for u in urls:
        html = urllib.request.urlopen(u, context=ctx).read().decode('utf-8')
        links = re.findall(r'<meta property=\"og:image\" content=\"(.*?)\"', html)
        if links: f.write(links[0] + '\n')
